integration-test/test_dummy_data.py

```diff
@@ -0,0 +1,185 @@
+# Note that this assumes that the KNMI test data is loader (using loader container)
+import logging
+import os
+from datetime import datetime
+from datetime import timezone
```

Comment:
I am not a fan of the file name. Maybe better to split it up in two?
`test_delete`
`test_polygon`

Also, why not test the polygon lookup on the actual KNMI data (like the lookup by station and parameter)? It saves a lot of setup code.